packages/superset-ui-time-format/test/factories/createD3TimeFormatter.test.js

```diff
@@ -1,32 +1,8 @@
 import createD3TimeFormatter from '../../src/factories/createD3TimeFormatter';
-import { PREVIEW_TIME } from '../../src/TimeFormatter';
-import { TimeFormats } from '../../src';
 
 describe('createD3TimeFormatter(config)', () => {
   it('requires config.formatString', () => {
```

Comment:
any reason to not just use `@ts-ignore` and have a single (`.ts`) file?

Reply:
This was written before I know about `@ts-ignore`. Will update.